data/db.js

```diff
@@ -2,10 +2,8 @@ require("dotenv").config(); //search for env variables
 const mongoose = require("mongoose");
 
 //set up db connection link
-const password = process.env.DB_ADMIN_PASSWORD;
-const dbname = "ucredit-db";
 const debug = process.env.DEBUG === "True";
-const URI = debug ? "mongodb://db:27017/debug" : `mongodb+srv://ucredit-admin:${password}@cluster0.ccsle.mongodb.net/${dbname}?retryWrites=true&w=majority`;
+const URI = debug ? "mongodb://db:27017/debug" : process.env.URI; 
```

Comment:
Please add comments here explaining the logic, for example: debug mode connects to local MongoDB, otherwise URI should strictly be the dev db URI unless in a prod environment

Reply:
TODO: document + comment